pkg/tpl/funcs.go

```diff
@@ -90,6 +91,55 @@ func Default(def interface{}, value interface{}) interface{} {
 	return value
 }
 
+func Marshal(value interface{}) string {
+	if value == nil {
+		return ""
+	}
+
+	v, err := json.MarshalIndent(value, "", "  ")
+	if err != nil {
+		return ""
+	}
+
+	return fmt.Sprintf("```json\n%s\n```", string(v))
+}
+
+func MarshalInline(value interface{}) string {
+	if value == nil {
+		return ""
+	}
+
+	v, err := json.Marshal(value)
+	if err != nil {
+		return ""
+	}
+
+	return fmt.Sprintf("`%s`", string(v))
+}
+
+func TableHeader(columns []string) string {
+	var header string
+	var divider string
+	for _, col := range columns {
+		header += fmt.Sprintf("| %s ", col)
```

Comment:
```suggestion
		header += fmt.Sprintf("|%s ", col)
```

Reply:
Your suggestion will just remove the space before heading, so it will look like this
```
|Name |Type |Description |
|------|------|-------------|
```